src/applications/check-in/day-of/pages/Error.jsx

```diff
@@ -10,22 +10,54 @@ const Error = () => {
   const selectError = useMemo(makeSelectError, []);
   const { error } = useSelector(selectError);
 
-  const message =
-    error === 'max-validation'
-      ? t('were-sorry-we-couldnt-match-your-information-please-ask-for-help')
-      : t(
-          'were-sorry-something-went-wrong-on-our-end-check-in-with-a-staff-member',
-        );
+  let messageText = '';
+  let alertType = '';
+  let header = '';
+
+  switch (error) {
+    case 'max-validation':
+      alertType = 'error';
+      header = t('we-couldnt-check-you-in');
+      messageText = t(
+        'were-sorry-we-couldnt-match-your-information-please-ask-for-help',
+      );
+      break;
+    case 'uuid-not-found':
+      // Shown when POST sessions returns 404.
+      alertType = 'info';
+      header = t('were-sorry-this-link-has-expired');
+      messageText = (
+        <Trans
+          i18nKey="trying-to-check-in-for-an-appointment--info-message"
+          components={[
+            <span key="bold" className="vads-u-font-weight--bold" />,
+            <va-telephone
+              key="53079"
+              data-testid="error-message-sms"
+              contact="53079"
```

Comment:
I checked and we're not using it anywhere else but even still do you think it might be worth putting the number in `appConstants`?

Reply:
I could it doesn't look like any of the other phone numbers are constants either. Should I add the others as well?

Reply:
hmm, I'm in favor, particularly for repeated ones like 8006982411 - what say you @brianseek ?